src/components/cartToggleButton/CartToggleButton.styled.ts

```diff
@@ -0,0 +1,33 @@
+import { Theme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+interface ButtonColorMapperProp {
+  $isInCart: boolean;
+  color: Theme['color'];
+}
+
+const buttonColorMapper = ({ $isInCart, color }: ButtonColorMapperProp) => {
+  if ($isInCart) {
+    return { backgroundColor: color.lightGray, color: color.black };
```

Comment:
이건 저도 헷갈려서 그러는데 아래처럼 `css`로 감싸지 않아도 잘 동작하나요?
```suggestion
    return css({ backgroundColor: color.lightGray, color: color.black });
```